### PR TITLE
TST: improve test coverage for `stats.mvsdist` and `stats.bayes_mvs`

### LINKED ISSUE #674: Statistics Review: bayes_mvs (Trac #147)
_Original ticket http://projects.scipy.org/scipy/ticket/147 on 2006-04-02 by @rkern, assigned to @rkern._

The function `bayes_mvs` in file source:trunk/Lib/stats/morestats.py needs review.

Please look over the StatisticsReview guidelines and add your comments below.

### PR DESCRIPTION
**Pull request by aeklant**
closes gh-674

### CODE REVIEW

File: scipy/stats/tests/test_morestats.py [hunk 1]
```diff
@@ -37,6 +37,24 @@
 
 
 class TestBayes_mvs(TestCase):
+    def test_basic(self):
+        data = [6, 9, 12, 7, 8, 8, 13]
+        mean, var, std = stats.bayes_mvs(data)
+        assert_almost_equal(mean.statistic, 9.0)
+        assert_allclose(mean.minmax, (7.1036502226125329, 10.896349777387467),
```
Comment (josef-pkt) [inline scipy/stats/tests/test_morestats.py]: Can you add a code comment for the origin of these numbers, regression test, other package, verified by hand, ...?

Comment (aeklant) [inline scipy/stats/tests/test_morestats.py]: I took it directly from the docstring example. Not sure what was the original source but I can verify by hand at least.

**Comment by rgommers** [comment]
@aeklant I've added a second example and fixed a few minor things in https://github.com/rgommers/scipy/tree/pr/5065. I've also squashed your 2 commits. Can you take over that branch here? Together with the things I checked in https://github.com/scipy/scipy/issues/674#issuecomment-127026124 I think that this is then ready to merge.

File: scipy/stats/tests/test_morestats.py (hunk 1 above)
Comment (rgommers) [inline scipy/stats/tests/test_morestats.py]: Took along this request in https://github.com/rgommers/scipy/commit/26cfec405d915aa22296043520fe8fe7ab1e8c12

**Comment by rgommers** [comment]
To take over the branch:

    $ git fetch rgommers
    $ git branch -D bayes_mvs  # make sure you have no new commits on this branch (they'll be gone)
    $ git checkout -b bayes_mvs rgommers/pr/5065  # now you've taken it over locally
    $ git push origin bayes_mvs -f  # need force-push because of squashed commits

**Comment by aeklant** [comment]
Alright, took over the branch with your additions. Is this all? I didn't quite get if there were any action items in the comment you mention.

**Comment by aeklant** [comment]
There is a check that fails because of the formatting in an example:

Expected:
    Std_dev(statistic=2.9724954732045084,
    ...     minmax=(1.7823367265645145, 4.9456146050146312))
Got:
    Std_dev(statistic=2.9724954732045084, minmax=(1.7823367265645143, 4.9456146050146295))

We have to either disable this check, or make the line non-compliant with PEP-8 (more than 80 characters in a line) for it to pass.

**Comment by rgommers** [comment]
The pep8 bot shouldn't complain, so let's just make it a single long line.

**Comment by rgommers** [comment]
That should indeed be all, no other comments from my side.

**Comment by aeklant** [comment]
Alright, it's done. Checks pass now.

**Comment by rgommers** [comment]
In it goes, thanks @aeklant

**Comment by argriffing** [comment]
This doctest is brutal!

**Comment by rgommers** [comment]
you mean too many significant digits?

fine to remove some I think

**Pull request merged**